### ISSUE TITLE
Issue with Setup

### ISSUE THREAD

**Issue by ZSN98** [jaabell/gmsh2opensees#3]
I am trying to do the install of g2o using "python setup.py install --user" and I get and error. I have attached the error given in the image. Any recommendation to solve this issue

![Setup](https://github.com/jaabell/gmsh2opensees/assets/125388039/ef870baf-2e82-49b8-b45b-b032aae93b33)

**Comment by ZSN98** [comment]
Also when I try to run I get this as the first error.

![image](https://github.com/jaabell/gmsh2opensees/assets/125388039/8141cd4e-3a83-4033-8862-f31b6dd328fe)

**Comment by jaabell** [comment]
Hi. Were you able to able to resolve this issue?

**Comment by ZSN98** [comment]
Yes the issue is still persisting. Unfortunately my programming skills are still being developed. The issue still persists where the setup is not completing.

**Comment by ZSN98** [comment]
Is there a way to manually import the library?

**Comment by ZSN98** [comment]
Dear Jaabell,

I have installed your program on Linux and I didn't encounter any of the previous problems.
In regards to the setup, after I tried doing the setup on Linux Alma9. I used it, but for some reason your file which imports opensees as ops didn't work I had to change the g2o_nodes_functions.py file: the change was on line 12, I changed it to import openseespy.opensees as ops.

I hope this gives you some insight for future updates and thanks again for such an amazing program.

**Comment by jaabell** [comment]
Hello,
Thanks for the feedback and suggestions. I'll keep them in mind.


Happy modeling!
JAA
--

José A. Abell, PhD
Ingeniero Civil
www.joseabell.com
